modules/drivers/redshift/test/metabase/driver/redshift_test.clj

```diff
@@ -1,10 +1,15 @@
 (ns metabase.driver.redshift-test
-  (:require [clojure.test :refer :all]
+  (:require [clojure.string :as str]
+            [clojure.test :refer :all]
+            [metabase.driver.sql-jdbc.execute :as execute]
             [metabase.plugins.jdbc-proxy :as jdbc-proxy]
-            [metabase.test
-             [fixtures :as fixtures]
-             [util :as tu]]
-            [metabase.test.data.datasets :refer [expect-with-driver]]))
+            [metabase.query-processor :as qp]
+            [metabase.test :as mt]
+            [metabase.test.data.datasets :refer [expect-with-driver]]
```

Comment:
I just wrote a long note about how we switched from Expectations to `clojure.test` and accidentally deleted it. So check your email and hopefully it's there so I don't have to type it out again. If not the TL;DR is we switched to `clojure.test` so don't use `expect*` for writing new tests (and bonus points for rewriting old ones to use the new style)

Reply:
removed use of expect in this test, will apply that going forward too